frontend/requests/analyse_hierarchy_request.css

```diff
@@ -0,0 +1,11 @@
+.analyse_request__description{
```

Comment:
the correct pattern is analyse-request__description. See those [examples](https://seesparkbox.com/foundry/bem_by_example).